ngraph/core/reference/include/ngraph/runtime/reference/matmul.hpp

```diff
@@ -56,7 +56,7 @@ namespace ngraph
                     }
                 }
 
-                std::vector<size_t> get_transpose_order(const Shape& input_shape)
+                static std::vector<size_t> get_transpose_order(const Shape& input_shape)
```

Comment:
why did you add static to this function? I propose to revert this change

Reply:
Otherwise, I received names conflict during linking. The same name is declared for transformations. This function must be internal

Reply:
Fixed.